app/models/proxy.rb

```diff
@@ -355,11 +355,20 @@ def save_and_deploy(attrs = {})
 
     analytics.track('Sandbox Proxy updated', analytics_attributes.merge(success: saved))
 
-    saved && deploy
+    if saved
+      success = deploy
+      analytics.track('Sandbox Proxy Deploy', success: success)
+    end
+
+    saved && success
```

Comment:
Remove this if https://github.com/3scale/porta/pull/1470/files#r354454921 of course.